x-pack/legacy/plugins/fleet/server/libs/agent_event.ts

```diff
@@ -49,3 +80,17 @@ export class AgentEventLib {
     });
   }
 }
+
+function assignPolicyId(agent: Agent, event: AgentEvent) {
+  event.policy_id = agent.policy_id;
+}
```

Comment:
I don't love this function as it is mutating an original object. as we are only calling it once from what I can tell (on L26 in this file), what do you think of doing the assignment without putting it in a function?